src/app/components/note/note.component.html

```diff
@@ -19,7 +19,9 @@ <h2 class="note-title">
         >delete</mat-icon
       >
     </h2>
-    <h4 class="note-date" *ngIf="selectedNote === note.id">{{ note.date }}</h4>
+    <h4 class="note-date" *ngIf="selectedNote === note.id">
+      {{ note.date | date }}
```

Comment:
Does the `date` variable exists?
![fox-mulder-xfiles](https://github.com/user-attachments/assets/a9bd75f4-fb07-40e0-8d58-34076204227b)

Reply:
I'm using date from DatePipe, actually

Reply:
Let's say that's the case.
Should the following code change still works:
![image](https://github.com/user-attachments/assets/5620be58-2f16-4b4e-a719-1eecc6e2f7b0)